ui/pages/settings/security-tab/reveal-srp-list/reveal-srp-list.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { Box } from '../../../../components/component-library';
+import SRPQuizModal from '../../../../components/app/srp-quiz-modal/SRPQuiz';
```

Comment:
Filename looks wrong now. Looking at other modals, it probably should be:
`components/app/srp-quiz-modal/srp-quiz-modal.tsx`